src/workflow.ts

```diff
@@ -33,7 +33,7 @@ export interface WorkflowParams {
 }
 
 export interface WorkflowConfig {
-  // TODO: add workflow config here.
+  registrationObject?: MethodRegistrationBase;
```

Comment:
Instead of passing in `MethodRegistrationBase` can we simply pass in the `schedulerConfig` object here?

Reply:
Could, but generically it surprises me that the workflow doesn't have access to the full registration object.  I would expect it to.  Overall, the data structures and code flow in this file deserve refactoring.  Perhaps separately.